docs/sphinx/sections/api/apidocs/libraries/dagster-aws.rst

```diff
@@ -98,9 +98,28 @@ Resources which surface SecretsManager secrets for use in Dagster resources and
 Pipes
 --------------
 
+Context Injectors
+^^^^^^^^^^^^^^^^^
+
+.. autoclass:: dagster_aws.pipes.PipesS3ContextInjector
+
+.. autoclass:: dagster_aws.pipes.PipesLambdaEventContextInjector
+
+Message Readers
+^^^^^^^^^^^^^^^
+
+.. autoclass:: dagster_aws.pipes.PipesS3MessageReader
+
+.. autoclass:: dagster_aws.pipes.PipesCloudWatchMessageReader
+   :members: consume_cloudwatch_logs
+
+Clients
+^^^^^^^
+
 .. autoclass:: dagster_aws.pipes.PipesLambdaClient
 
 .. autoclass:: dagster_aws.pipes.PipesGlueClient
+   :members: run
```

Comment:
@erinkcochran87 any idea why this method doesn't show up in the final API docs? same as `:member:` above

Reply:
It took me awhile but I finally figured it out

In `dagster-aws.rst`:

```
.. autoclass:: dagster_aws.pipes.PipesGlueClient
   :noindex:
   :members: run
```

I think the `noindex` is necessary because there may be other methods with this name. Sphinx should be throwing an error when it encounters a duplicate, but who knows.

And, in `pipes.py`:

```python
from dagster._annotations import experimental, public

...

    @public
    def run(
```

I tried this with the `@experimental` decorator as well, but Sphinx didn't pick it up.

Reply:
Thank you so much! That's some arcane knowledge 🤪